### ISSUE TITLE
W3C validator fails inside Reith

### ISSUE THREAD

**Issue by mattwynne** [cucumber-ltd/bbc-a11y#2]
@IanPouncey has reported:

> I've checked out the latest bbc-accessibility-standards-checker, I'm currently experiencing a problem with it though. The tests run, but fails on connecting to the W3C:
>
> ```
> When I submit the page to the W3C Markup Validation Service # features/step_definitions/steps.rb:1
>      Operation timed out - connect(2) for "validator.w3.org" port 80 (Errno::ETIMEDOUT)
>      ./features/support/world.rb:19:in `validate'
>      ./features/step_definitions/steps.rb:2:in `/^I submit the page to the W3C Markup Validation Service$/'
>      features/02_validation.feature:35:in `When I submit the page to the W3C Markup Validation Service'
>```
>I'm able to curl www.bbc.co.uk from the same terminal session, but I'm going to try it at home off the Reith network to make sure that's not the issue.

**Comment by mattwynne** [comment]
At the meeting / hack session yesterday the value of this story was questioned, since it's proving costly to implement and will always require an out-of-network round-trip.

**Comment by mattwynne** [comment]
Decided on yesterday's call to put this problem out of scope for now. Teams can ignore it using the `skip_scenario` configuration.
